devtools-ci-index.yaml

```diff
@@ -4445,15 +4295,6 @@
             saas_git: saas-errortracking
             prj_name: sentry-preview
             timeout: '20m'
-        - '{ci_project}-{git_repo}-f8planner':
-            git_organization: fabric8-ui
-            git_repo: fabric8-planner
-            ci_project: 'devtools'
-            ci_cmd: '/bin/bash cico_run_tests.sh'
-            registry: 'quay.io'
-            image_name: 'openshiftio/fabric8-ui-fabric8-planner'
-            timeout: '1h'
-            discarder_days: 30
         - 'devtools-test-end-to-end-{test_url}-planner-api-test':
```

Comment:
@ldimaggi @pmacik can we remove this job?

Reply:
@ppitonak Yes.

Reply:
@MatousJobanek it was already removed in #1000